std/root.bass

```diff
@@ -214,23 +214,29 @@
     [] z
     [x & xs'] (foldl f (f z x) xs')))
 
-(provide (append)
-  (defn append1 [xs ys]
+(provide (concat)
+  (defn concat1 [xs ys]
     (case xs
       [] ys
-      [x & xs'] (cons x (append1 xs' ys))))
+      [x & xs'] (cons x (concat1 xs' ys))))
 
   ; joins all given lists into one list
   ;
-  ; => (append [1] [2 3] [4 5 6])
-  (defn append xss
-    (foldl append1 [] xss)))
+  ; => (concat [1] [2 3] [4 5 6])
+  (defn concat xss
+    (foldl concat1 [] xss)))
+
+^{:deprecated "Use (concat) instead."}
+(def append concat)
+
+(meta append)
+; => {:doc "concatenates things" :file <fs>/history :line 1 :column 0 :deprecated "Use (concat) instead."}
```

Comment:
```suggestion
```

This bit isn't needed, I just included it in my comment for demonstration purposes. :)